emar.py

```diff
@@ -11,14 +11,15 @@
 
 import sys
 
-from tools import shared, building
+from tools import shared
 
 
 #
 # Main run() function
 #
 def run():
-  return shared.run_process([shared.LLVM_AR] + building.get_command_with_possible_response_file(sys.argv[1:]), stdin=sys.stdin, check=False).returncode
+  cmd = [shared.LLVM_AR] + sys.argv[1:]
+  return shared.run_process(cmd, stdin=sys.stdin, check=False).returncode
```

Comment:
it does seem natural to use `building.get_command_with_possible_response_file` - if I were refactoring this I would want to do that. maybe a comment here why it can't work, or why it isn't necessary? (I still don't understand why not)

Reply:
but we know that the command line here (by definition) does not need a response file..  also doing that alone would be broken because sys.argv here might already have response file.. so it only makes sense if you have somehow already opened and expanded any incoming response file, which we don't do by default (at least not prior to #13954)

Reply:
Oh, you mean that if it worked to call this script with those args, then the number of args is reasonable anyhow? Sgtm.